content/events/bmt-2024.md

```diff
@@ -0,0 +1,333 @@
+    <tr>
+      <th>% of Score</td>
+      <td>30%</td>
+      <td colspan="2">40%</td>
+      <td>30%</td>
+    </tr>
+    <tr>
+      <th>Important Scoring Notes</td>
+      <td>N/A</td>
+      <td colspan="2">
+        If teams have less than 4 members, the remaining scores will be counted as 0. The General Test will only count for half as many points as the Focus combinations
+      </td>
+      <td>Teams are live scored, and there will be a displayed scoreboard</td>
+    </tr>
+    <tr>
+      <th>Collaboration?</td>
+      <td>Yes</td>
+      <td>No</td>
+      <td>No</td>
+      <td>Yes</td>
+    </tr>
+  </tbody>
+</table>
+
+## Acceptable Answers
+
+Students are responsible for familiarizing themselves with the
+[acceptable test formatting document](/assets/answer-formats.pdf) and
+[common notions document](/assets/common-notions.pdf) for details on how their
+answers should be submitted. Answers not submitted in the listed formats will be
+graded as incorrect, even if equivalent.
+
+## Activities
+
+After lunch, we will be hosting optional student activities that promote
+recreational math. Students that choose to participate will have the opportunity
+to win amazing prizes! Below are the activities we are expecting to run:
+
+### Estimathon
+
+A team-based contest that combines trivia, game theory, strategy, and
+mathematical thinking. Teams of ~3 will face a tight time limit to work on a set
+of 13 Fermi-style estimation problems.
+
+### Fraudulent Math
+
+Attention all students! Mathematics is under attack by people who claim that we
+need “rigor” and “logic” to justify our answers. Join us for an event of
+Fraudulent Math, where students will compete to come up with the most
+outlandish, mind bending proofs of false statements and to create impossible
+constructions.
+
+### Integration Bee
+
+The hypest activity at BMT: compete head to head against the best integrators
+around! Students will face off in a tournament style bracket, solving difficult
+integrals involving some clever tricks, to determine who will be the champion of
+all integrators.
+
+### Origami
+
+Do you want to bring home a cute souvenir from BMT? Come join us for a fun
+origami session! We will be offering multiple paper patterns and instructions,
+and teaching from the basics of origami. No prior craft experience is needed!
+
+## Frequently Asked Questions
+
+### Are calculators or other reference material allowed?
+
+Graph paper and calculators are prohibited, but protractors, rulers, and
+compasses are permitted. Blank white scratch paper will be provided for all
+participants. All answers must be exact, reduced, and simplified. Illegible
+answers will not be graded. Cheating in any form will not be tolerated, and
+failure to comply with all rules may result in immediate and irreversible
+disqualification.
+
+### Can middle school students compete? What if I don’t have a team?
+
+Yes! BMT is open to any student in 8th grade or below, including those who
```

Comment:
BMT is open to students in 12th grade or below, not 8th grader or below

Reply:
wtf LOL